builddefs/common_features.mk

```diff
@@ -573,11 +574,7 @@ ifeq ($(strip $(BACKLIGHT_ENABLE)), yes)
         OPT_DEFS += -DBACKLIGHT_CUSTOM_DRIVER
     else
         SRC += $(QUANTUM_DIR)/backlight/backlight_driver_common.c
-        ifeq ($(strip $(BACKLIGHT_DRIVER)), pwm)
-            SRC += $(QUANTUM_DIR)/backlight/backlight_$(PLATFORM_KEY).c
-        else
-            SRC += $(QUANTUM_DIR)/backlight/backlight_$(strip $(BACKLIGHT_DRIVER)).c
-        endif
+        SRC += backlight_$(strip $(BACKLIGHT_DRIVER)).c
```

Comment:
Not a fan of having a relative path.  They tend to generate build objects in less than ideal places.

But not sure there is a better way to handle this, other than with a bunch of if's

Reply:
Yeah, I wasn't completely happy with that, but I think I can make it slightly better.

Reply:
Change isn't great, but at least build artifacts should be handled better. 👍🏻